angular.json

```diff
@@ -22,7 +28,8 @@
             "aot": true,
             "assets": [
               "src/favicon.ico",
-              "src/assets"
+              "src/assets/logos",
+              "src/assets/md-files"
             ],
```

Comment:
exclude **src/assets/example-sources** so it doesn't get processed by typescript/javascript compiler.

Reply:
Instead of explicitly enumerating all the sub-folders (which will cause the need to always update this whenever a new folder will be added to `src/assets`), why not just say: take everything from `src/assets` folder, except `example-sources` folder? Something like this:
`{
                "glob": "**/*",
                "input": "src/assets",
                "output": "/assets",
                "ignore": ["examples-sources/*"]
}`

Reply:
yes, that would be better.  

Does the {} expression directly replace the "src/assets" line (line 25)?

Reply:
Correct. Assets array should look like this:
`
"assets": [
              "src/favicon.ico",
               { "glob": "**/*", "input": "src/assets", "output": "/assets", "ignore": ["examples-sources/*"] }
            ]
`